.travis.yml

```diff
@@ -16,7 +16,8 @@ matrix:
       - cd ../
       - npm init ember-app test-app
       - cd test-app
-      - "node_modules/.bin/ember -v | grep -q 'ember-cli: 3.6'"
+      - "export CURRENT_VERSION=`curl -H 'Authorization: token $GITHUB_TOKEN' https://api.github.com/repos/ember-cli/ember-cli/releases/latest | jq '.tag_name' | grep -o '\\d\\+\\.\\d\\+\\.\\d\\+'`"
+      - "node_modules/.bin/ember -v | grep -q 'ember-cli: $CURRENT_VERSION'"
```

Comment:
this is starting to look quite complicated. would it be worth it to use a proper JS test runner to run those commands? I assume that might make it possible to run the tests locally too? 🤔

Reply:
If we want to run the ember tests we can literally just add the line `npm test` or `yarn test` to the end of this. My reasoning for **not** doing this was that running the ember tests doesn't really test the _point_ of this repo. All we really need to know is what version `ember -v` reports to us.

I'm also not 100% sure this is all that complicated compared to building test-runner infrastructure 🤔 I could maybe clean this "oneliner" up a bit to make it less complicated looking if that would help?

Reply:
But these tests need to be run in the context of the **newly created ember app**, not in the context of the project itself. That's why I didn't originally create a test script that could be run with `npm test` in the first place 😞 

I really don't think that these sorts of tests belong in a JS test runner because they are essentially **infrastructure** tests that want to test something that is as close as possible to running `npm init ember-app test-project` directly

Reply:
that's why I would use something like `execa` to run `npm init ember-app test-project` from within the test runner. basically, instead of using shell commands we use JS to do the same thing, and we wrap it in a test runner to get nicer feedback.

I might be over-engineering this though... 😅

Reply:
I wasn't really going for a "nice feedback" test runner scenario here 😖 all I really want is a single smoke test in both npm and yarn to make sure that when someone runs this they get what **we** expect them to get. 

For example, I tweeted about this functionality a while back and until I noticed that `npm init ember-app test-project` didn't automatically create an up-to-date project for me I had no clue that this was failing for everyone without ember installed globally 😭 

These "messy" tests are a nice-to-have because there was literally nothing there before. I am happy to do whatever to get this landed but I don't think adding a test runner in JS helps with the goal to make this "simpler"

Reply:
yeah, makes sense, we can always iterate on it if needed. having this as a basic smoke test is already much better than what we had before :)